src/interfaces/earnings.ts

```diff
@@ -129,32 +129,36 @@ export class EarningsInterface<C extends ChainId> extends ServiceInterface<C> {
         return {
           assetAddress: getAddress(assetPosition.vault.id),
           balanceUsdc,
-          earned: earningsUsdc.toFixed(0)
+          earned: earningsUsdc.toFixed(0),
         };
       })
     );
 
-    const assetAddresses = assetsData.map(assetData => assetData.assetAddress);
+    const assetAddresses = assetsData.map((assetData) => assetData.assetAddress);
     const apys = await this.yearn.services.vision.apy(assetAddresses);
 
-    const totalEarnings = assetsData.map(datum => new BigNumber(datum.earned)).reduce((sum, value) => sum.plus(value));
-    const holdings = assetsData.map(datum => new BigNumber(datum.balanceUsdc)).reduce((sum, value) => sum.plus(value));
+    const totalEarnings = assetsData
+      .map((datum) => new BigNumber(datum.earned))
+      .reduce((sum, value) => sum.plus(value));
+    const holdings = assetsData
+      .map((datum) => new BigNumber(datum.balanceUsdc))
+      .reduce((sum, value) => sum.plus(value));
```

Comment:
We had the `"printWidth": 120` in package.json prettier config but it wasn't being taken into consideration ,that's fixed with this PR so prettier will try and wrap them at 120 chars length